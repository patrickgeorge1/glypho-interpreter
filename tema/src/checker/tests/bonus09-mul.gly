"""9&&&6%%%:O))dD>>_hwww