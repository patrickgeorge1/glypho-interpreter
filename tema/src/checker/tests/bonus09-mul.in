-135478
6532
3215557
