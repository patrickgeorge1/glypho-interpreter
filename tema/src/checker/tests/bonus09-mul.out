-3286868818577074
